packages/discord.js/src/client/Client.js

```diff
@@ -512,6 +515,21 @@ class Client extends BaseClient {
     if (typeof options.failIfNotExists !== 'boolean') {
       throw new DiscordjsTypeError(ErrorCodes.ClientInvalidOption, 'failIfNotExists', 'a boolean');
     }
+    if (options.allowedMentions && typeof options.allowedMentions !== 'object') {
+      throw new DiscordjsTypeError(ErrorCodes.ClientInvalidOption, 'allowedMentions', 'an object');
+    }
```

Comment:
This should have a default.

Reply:
What should it default to? I was thinking an empty object but there might be a better default.

Reply:
For this to be a non-breaking change the default needs to be `undefined`. So the check should allow that too.

Reply:
Since it's already `undefined` by default, does [`c3dbd35`](https://github.com/TetieWasTaken/discord.js-fork/commit/c3dbd35bcffe4cf3c18d9d18941ff00032d0fb3c) fix this?

Reply:
From @Qjuh's comment, I guess this is currently fine as-is (original review). It'll still allow `null`, though.